docs/modules/ROOT/pages/runbooks/APPUiOCloudReportingDatabaseTenantMappingFailed.adoc

```diff
@@ -0,0 +1,17 @@
+= Alert rule: APPUiOCloudReportingDatabaseTenantMappingFailed
+
+include::partial$runbooks/contribution_note.adoc[]
+
+== icon:glasses[] Overview
+
+APPUiO Cloud Reporting failed while trying to add a tenant target mapping from billing metrics to the reporting database.
+This can lead to problems with the invoice generation but shouldn't be time critical.
```

Comment:
nb: Some additional information on when this is time critical might be helpful.

```suggestion
This can lead to problems during the monthly invoice generation.
```